cli/script/command-executor.ts

```diff
@@ -918,6 +923,73 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
         });
 }
 
+export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void> => {
+    var platform: string = command.platform.toLowerCase();
+    var platformFolder: string = path.join(process.cwd(), "platforms", platform);
+    var platformCordova: string = path.join(platformFolder, "cordova");
+    var outputFolder: string;
+    var preparePromise: Promise<void>;
+
+    if (platform === "ios") {
+        outputFolder = path.join(platformFolder, "www");
+    } else if (platform === "android") {
+        outputFolder = path.join(platformFolder, "assets", "www");
+    } else {
+        throw new Error("Platform must be either \"ios\" or \"android\".");
+    }
+
+    try {
+        var tempPrepareContents = `
```

Comment:
I thought we didn't add type annotations to literals? Are we explicitly typing everything now? I added it, but just curious.